tests/page-objects/forms/generic-form.wdio.page.js

```diff
@@ -0,0 +1,118 @@
+const nameField = () => $('#report-form form [name="/data/name"]');
```

Comment:
It seems like this page object was just pulled over and some portions were updated.  Can we remove what isn't being used to avoid having a mix of protractor syntax and wdio?

Reply:
It's also a good time to remove the native suffix as well.